files/class-vip-filesystem-api-cache.php

```diff
@@ -0,0 +1,81 @@
+<?php
```

Comment:
Minor: Let's rename this to `class-api-cache.php` to keep the naming consistent. (It's also not specific to the vip-filesystem either.)

Reply:
That is true. I'll change it then